website/package.json

```diff
@@ -6,8 +6,8 @@
     "deploy": "docusaurus deploy"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.39",
-    "@docusaurus/preset-classic": "2.0.0-alpha.39",
+    "@docusaurus/core": "2.0.0-alpha.40",
+    "@docusaurus/preset-classic": "2.0.0-alpha.40",
```

Comment:
Outta curiosity, is there a specific reason that these are getting version bumps? Might be better to branch out these version bumps to a separate PR.